test/bin/scenario.sh

```diff
@@ -1102,36 +1118,46 @@ run_gingko_tests() {
     fi
     record_junit "${vmname}" "oc_installed" "OK"
 
-    # Check/get openshift-tests-binary
-    if ! "${ROOTDIR}/scripts/fetch_tools.sh" "ginkgo"; then
-        record_junit "${vmname}" "build_test_binary" "FAILED"
+    # Wait for MicroShift to be ready
+    wait_for_microshift_to_be_ready "${vmname}"
```

Comment:
This is not just setting up oc and kubeconfig, right? Kubeconfig is available as soon as MicroShift starts, you dont need to wait until its ready. It can be done earlier.